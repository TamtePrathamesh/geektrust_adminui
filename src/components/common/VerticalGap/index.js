import React from 'react';
import PropTypes from 'prop-types';
import style from './style.module.css';

const VerticalGap = ({ height = 0 }) => {
  return <div className={style.gap} style={{ height: `${height}px` }}></div>;
};

VerticalGap.propTypes = {
  height: PropTypes.string,
};

export default React.memo(VerticalGap);
