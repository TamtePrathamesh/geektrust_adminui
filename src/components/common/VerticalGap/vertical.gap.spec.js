import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import VerticalGap from './index';

describe('VerticalGap', () => {
  it('should match snapshot', () => {
    const renderer = new ShallowRenderer();
    expect(renderer.render(<VerticalGap />)).toMatchSnapshot();
  });
  it('should match snapshot when given height', () => {
    const renderer = new ShallowRenderer();
    expect(renderer.render(<VerticalGap height="15" />)).toMatchSnapshot();
  });
});
