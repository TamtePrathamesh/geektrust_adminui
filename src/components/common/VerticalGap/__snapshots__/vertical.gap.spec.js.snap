// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerticalGap should match snapshot 1`] = `
<div
  className="gap"
  style={
    Object {
      "height": "0px",
    }
  }
/>
`;

exports[`VerticalGap should match snapshot when given height 1`] = `
<div
  className="gap"
  style={
    Object {
      "height": "15px",
    }
  }
/>
`;
